### PR TITLE
Fix minor grammar or spelling errors

### PR DESCRIPTION
**Pull request by dqdinh**

### CODE REVIEW

File: site/src/main/tut/tutorial/tutorial.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ anything, instead it will return an `IO` instance that encapsulates all the
 side effects involved (opening/closing files, reading/writing content), that way
 _purity_ is kept.  Only when that `IO` instance is evaluated all those
 side-effectful actions will be run. In our implementation the `IO` instance will
-return the amount of bytes copied upon execution, but this is just a design
+return the amount of bytes copied upon execution, but this is just a desrrrrrign
```
Comment (alexandru) [inline site/src/main/tut/tutorial/tutorial.md]: What is `desrrrrrign`?

File: site/src/main/tut/tutorial/tutorial.md [hunk 2]
```diff
@@ -292,7 +292,7 @@ iteration.
 We are making progress, and already have a version of `copy` that can be used.
 If any exception is raised when `transfer` is running, then the streams will be
 automatically closed by `Resource`. But there is something else we have to take
-into account: `IO` instances execution can be **_canceled!_**. And cancellation
+into account: `IO` instances execution can be **_cancelled!_**. And cancellation
```
Comment (alexandru) [inline site/src/main/tut/tutorial/tutorial.md]: "canceled" is correct too, being the American spelling of "cancelled", please revert.

File: site/src/main/tut/tutorial/tutorial.md (hunk 1 above)
Comment (dqdinh) [reply site/src/main/tut/tutorial/tutorial.md]: whoops :) - I fat fingered that one.

File: site/src/main/tut/tutorial/tutorial.md (hunk 2 above)
Comment (dqdinh) [reply site/src/main/tut/tutorial/tutorial.md]: my apologies - sorry for the white noise

**Comment by LukaJCB** [comment]
@dqdinh "We consider opening a stream to be a side-effect action" this is still a valid fix :P

**Pull request closed without merging**